addons/mass_mailing/views/res_config_settings_views.xml

```diff
@@ -53,6 +53,17 @@
                                     </div>
                                 </div>
                             </div>
+                            <div class="col-md-6 o_setting_box col-xs-12" name="mass_mailing_reports_setting_container">
+                                <div class="o_setting_left_pane" title="Send a report to the mailing responsible one day after the mailing has been sent.">
```

Comment:
I think this string should also be translated
![screenshot_spect](https://user-images.githubusercontent.com/54316226/146737524-e5bbcbff-f3cf-4fff-b311-f76218f2e8af.png)